src/core/currency/currency-pixie.ts

```diff
@@ -21,5 +23,28 @@ export const currency: TamePixie<RootProps> = combinePixies({
       walletState: props.state.currency.wallets[walletId],
       walletOutput: props.output.currency.wallets[walletId]
     })
-  )
+  ),
+
+  pluginUpdater(input: ApiInput) {
+    let lastInfo: InfoCacheFile | undefined
+
+    return async () => {
+      const { infoCache, plugins } = input.props.state
+
+      for (const pluginId of Object.keys(plugins.currency)) {
+        const plugin = plugins.currency[pluginId]
+        const newPayload = infoCache.corePlugins?.[pluginId] ?? {}
+        const oldPayload = lastInfo?.corePlugins?.[pluginId] ?? {}
+
+        if (
+          plugin.updateInfoPayload != null &&
+          lastInfo != null &&
+          !matchJson(oldPayload, newPayload)
+        ) {
+          await plugin.updateInfoPayload(newPayload)
+        }
+      }
+      lastInfo = infoCache
+    }
+  }
```

Comment:
This don't call updateInfoPayload on the first invocation, so I'm guessing that the first invocation is right after initializing the plugin with the info payload?

Reply:
Right, that was the intent. The plugins get the initial value at boot-up, so we only need to update them if something has changed since then.